### PR TITLE
Arrivals List, Separation Assurance, and more

### PR DESCRIPTION
**Pull request by alphadelta332**
## Summary
Introduction of Arrivals List usage SOPs, as well as other changes surrounding Separation Assurance and Sequencing

## Changes
**Fixes**:
- Typos/Formatting in **Annotations** page

**Changes**:
- Moved _Sequencing standards_ to **Surveillance** page

**Additions**:
- **Arrivals List** SOPs to **Sequencing** page
- Many changes/additions to **Separation Assurance** page, including tips on:
    - BRL
    - Closest Approach Tool
    - System Estimates
    - Level Assignment

### CODE REVIEW

**Review by tylerthetiletiler — APPROVED** [review]

File: docs/controller-skills/sequencing.md [hunk 1]
```diff
@@ -8,12 +8,88 @@ title: Sequencing
...
+The Arrivals List window is a sequencing management tool that can be used side-by-side with, or in-lieu of, [Maestro](../maestro).
+
+It is a **global** window that can be viewed and manipulated by Tower, TCU, and Enroute Controllers.
+
+The responsibility for managing the window falls on the:
+
+- **Flow** Controller
+    - This is delegated to the **Approach** Controller when Flow is Offline
+- **ADC** Controller at [Procedural Towers](../../aerodromes/procedural/)
+
```
Comment (mattkelly4) [inline docs/controller-skills/sequencing.md:27]: Could we throw a note in here that the ADC (or TMA if top down) controller should move aircraft back above the bar as they land?

File: docs/controller-skills/sequencing.md [hunk 2]
```diff
@@ -8,12 +8,88 @@ title: Sequencing
...
 
+## Arrivals List
+The Arrivals List window is a sequencing management tool that can be used side-by-side with, or in-lieu of, [Maestro](../maestro).
+
+It is a **global** window that can be viewed and manipulated by Tower, TCU, and Enroute Controllers.
+
+The responsibility for managing the window falls on the:
+
+- **Flow** Controller
+    - This is delegated to the **Approach** Controller when Flow is Offline
```
Comment (mattkelly4) [inline docs/controller-skills/sequencing.md:25]: Ideally the TMA guy would do it, but we don't really teach that to S3s. Maybe we could say 'Approach or surrounding Enroute controller'

File: docs/controller-skills/sequencing.md (hunk 1 above)
Comment (alphadelta332) [reply docs/controller-skills/sequencing.md:27]: Could do, but is it required? I believe the strip disappears when the aircraft disconnects from the network, is that sufficient? I believe IRL it disappears when the plane lands so i guess there's a bit more of a gap there if we wanna do that

File: docs/controller-skills/sequencing.md (hunk 2 above)
Comment (alphadelta332) [reply docs/controller-skills/sequencing.md:25]: Yeah my thinking is, it's fine if people want to not use it due to lack of training, very quiet airport, etc. But if they want to use it, the info is there in the SOPs

In my opinion, it's not of much value to be managed by a surrounding enroute controller. At YMML and YSSY, there are multiple surrounding enroute controllers. And even at YBBN, YPAD, and YPPH where there's only 1, the Enroute controller doesn't really have a picture of how they conflict in the sequence, they only care about separating aircraft through the same FF.

So yeah, my thoughts are, the actions affect, and are for the benefit of, the approach/flow controller. Therefore, it should be managed by them

**Review by mattkelly4 — APPROVED** [review]

**Review by LeGiraffe — APPROVED** [review]

**Pull request merged**